Automattic/wp-calypso
# Sites Management Page: Move to `/sites` and release to all users

**This PR is to prepare the release ahead of time. Don't merge until release time!**
The idea is to get this tested and approved so that it's one less thing to do on release day.

### Proposed Changes

* Enables the `build/sites-dashboard` flag in production and staging
* Take over the `/sites` URL.
* Any traffic that would have gone to `/sites` now goes to `/home` (the UI there is identical)
* Update server tests to reflect route changes

The section "group" and module are still called "sites-dashboard". This is to disambiguate it from the main Calypso dashboard, which is already called the "sites" group and there are already lots of sections that use it. That means that CSS rules like `body.is-group-sites-dashboard` stay as they currently are.

### Testing Instructions



* `/sites` now displays the SMP
* Create a new site from the SMP then click "Back to My Sites". You should still land back on the SMP.
* Check that you can navigate to the SMP from the site switcher.
* Disable the `build/sites-dashboard` flag in development and try the above steps again. It should match current production behaviour except that some URLs have been changed from `/sites` to `/home`

### Pre-merge Checklist






Related to #66727
